config/tasklists/learn-to-drive-a-car.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "Learn to drive a car: step by step",
   "base_path": "/learn-to-drive-a-car",
-  "description": "Check what you need to do to learn to drive.",
+  "description": "<p>Check what you need to do to learn to drive.</p>",
```

Comment:
This might be an indicator that we want to have this as a markdown field in our brave new publisher world?  (No action required)